stubs/fpdf2/fpdf/drawing.pyi

```diff
@@ -11,8 +11,8 @@ from .syntax import Name, Raw
 
 __pdoc__: dict[str, bool]
 
-def force_nodocument(item): ...
-def force_document(item): ...
+def force_nodocument(item) -> item: ...
+def force_document(item) -> item: ...
```

Comment:
The names we use in type annotations need to be names that are already defined in the global namespace. `item` here is not defined in the global namespace.

To say "this function returns the same thing that we put into the function", we can use a `TypeVar` (_type variable_). The docs for `TypeVar` are here: https://docs.python.org/3/library/typing.html#typing.TypeVar.

For example, in this case we'd use `TypeVar` like this:

```python
_T = TypeVar("_T")
def force_document(item: _T) -> _T: ...
```

The fact that the `item` parameter and the return type are annotated with the same type variable indicates to a static type checker that the same type is returned as the type that was passed in.

You'll need to add `from typing import TypeVar` at the top of the file.

Reply:
@AlexWaygood Yups thanks, it makes sense. 👍